src/node/build/index.ts

```diff
@@ -213,8 +214,10 @@ export async function build(options: BuildConfig = {}): Promise<BuildResult> {
       // Vue templates are compiled into js and included in chunks.
       createReplacePlugin(
         {
-          'process.env.NODE_ENV': '"production"',
-          'process.env.': `({}).`,
+          'process.env.': `(${JSON.stringify({
+            ...env,
+            NODE_ENV: 'production'
+          })}).`,
```

Comment:
Oh, does this now handle `process.env["NODE_ENV"]` and `Object.keys(process.env)` when building it?

Reply:
sorry, i just missed part of the information for this issue. It can be solved now.

Reply:
Thank you very much! 🎉